ansible/tasks/provision/delivery.yaml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Provision a persistent volume for Kafka
+  ec2_vol:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ region }}"
+    zone: "{{ region }}a"
+    name: "{{ stack_name }}-kafka"
+    volume_size: 10
+    volume_type: gp2
+    tags:
+      environment: "{{ environment_type }}"
+      teamDL: "universal.publishing.platform@ft.com"
+      systemCode: "{{ platform }}"
+  register: ec2_vol_del_kafka
+
+- name: EBS volume details
+  debug:
+    msg: "EBS Volume ID for delivery: '{{ ec2_vol_del_kafka.volume_id }}'"
+
+- name: Provision a persistent volume for Mongo
+  ec2_vol:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    region: "{{ region }}"
+    zone: "{{ item }}"
+    name: "{{ stack_name }}-mongo-{{ item }}"
+    volume_size: 50
+    volume_type: gp2
+    tags:
+      environment: "{{ environment_type }}"
+      teamDL: "universal.publishing.platform@ft.com"
+      systemCode: "{{ platform }}"
+  with_items: 
+    - "{{ region }}a"
+    - "{{ region }}b"
+    - "{{ region }}c"
+  register: ec2_vol_del_mongo
+
+
+- debug: var=ec2_vol_del_mongo
```

Comment:
It would be nice to have have a single debug message with all the provisioned things. It would help us put the right stuff in the global-config map.